src/libraries/Microsoft.PowerFx.Core/Localization/Strings.cs

```diff
@@ -160,7 +160,9 @@ internal static class TexlStrings
         public static StringGetter RandBetweenArg2 = (b) => StringResources.Get("RandBetweenArg2", b);
 
         public static StringGetter AboutNow = (b) => StringResources.Get("AboutNow", b);
+        public static StringGetter AboutUTCNow = (b) => StringResources.Get("AboutNow", b);
```

Comment:
Is this supposed to use the same resource string as the normal AboutNow function?